web-src/src/components/PromptResultCard.js

```diff
@@ -149,7 +150,8 @@ export function PromptResultCard({ result, ...props }) {
         ToastQueue.positive('Feedback sent', { timeout: 1000 });
       })
       .catch((error) => {
-        ToastQueue.negative('Failed to send feedback. Error code: IS-ERROR', { timeout: 1000 });
+        const errorResponse = getErrorResponse(error);
```

Comment:
Since the UI only requires displaying an error message, we can call this function where the error object is generated, rather than sending the full error metadata to the UI and making it extract the actual error message.

Reply:
The concept is straightforward:
- Since the UI can only show a message, is there a need to supply it with additional error-related metadata that it doesn't use?
- To generate an error message, we need context and specific details about the error condition. These are known at the point where the error is handled and the application specific error object is thrown. Once the message is crafted there, there's no need to pass any extra details with the error object further down the application (like origin, etc).
cc @roribio